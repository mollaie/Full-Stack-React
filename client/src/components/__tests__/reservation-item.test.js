import React from "react";
import renderer from "react-test-renderer";
import ReservationItem from "../reservation-item";

describe("reservation-item test", () => {
  it("reservation-item should match snapshot", () => {
    const component = renderer.create(<ReservationItem />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
