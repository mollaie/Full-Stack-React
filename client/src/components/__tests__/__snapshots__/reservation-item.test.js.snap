// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reservation-item test reservation-item should match snapshot 1`] = `
<div
  className="reservation-item"
>
  <p
    className="title"
  />
  <p
    className="status red"
  />
  <button
    className="btn btn-danger"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Delete
  </button>
</div>
`;
